File: Secao06 - Estruturas de Decisão/Exercicios em Python/Exercicio10.py
# -*- coding: utf-8 -*-
"""
Infantil A = 5 a 7 anos.
Infantil B = 8 a 11 anos.
Juvenil A = 12 a 13 anos.
Juvenil B = 14 a 17 anos.
Adulto = + 18 anos.
"""
print("Classificação em categorias.\n")
idade = int(input("Informe a sua idade para classificação: "))

if idade >= 5 and idade <= 7:
    print("Infantil A.")
elif idade >= 8 and idade <= 11:
    print("Infantil B.")
elif idade >= 12 and idade <= 13:
    print("Juvenil A.")
elif idade >= 14 and idade <= 17:
    print("Juvenil B.")
elif idade >= 18:
    print("Adulto.")
else:
    print("Erro! Idade inferior a 5 anos.")

File: Secao06 - Estruturas de Decisão/Exercicios em Java/Exercicio10.java
package br.com.udemy.secao06;

import java.util.Scanner;

public class Exercicio10 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		int idade;
		
		// Infantil A = 5 a 7 anos
		// Infantil B = 8 a 11 anos
		// Juvenil A = 12 a 13 anos
		// Juvenil B = 14 a 17 anos
		// Adultos +18 anos
		
		System.out.print("Informe sua idade para classificarmos: ");
		idade = teclado.nextInt();
		
		if(idade >= 5 && idade <= 7) {
			System.out.println("Infantil A");
		}else if (idade >= 8 && idade <= 11){
			System.out.println("Infantil B");
		}else if (idade >= 12 && idade <= 13){
			System.out.println("Juvenil A");
		}else if (idade >= 14 && idade <= 17){
			System.out.println("Juvenil B");
		}else if (idade >= 18){
			System.out.println("Adulto");
		}else{
			System.out.println("Idade não aceita. Minimo 5 anos!");
		}
		teclado.close();
	}

}
